.github/workflows/npm-publish.yml

```
@@ -0,0 +1,68 @@
+
+      # Create PR & publish to NPM
+      - name: Create release Pull Request or publish to NPM
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          publish: yarn release
+          commit: "ci(changesets): version packages"
+          title: "ci(changesets): version packages"
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)